src/agent/agent.c

```diff
@@ -1639,7 +1927,7 @@ static bool agent_connect(Agent *agent) {
                 return false;
         }
 
-        hirte_log_infof("Agent '%s' connection attempt %d: success", agent->name, agent->connection_retry_count);
+        hirte_log_infof("Connected to manager as '%s'", agent->name);
```

Comment:
You should change the grep in e2e 
https://github.com/containers/hirte/blob/main/.github/workflows/e2e.yml#L71-L73
https://github.com/containers/hirte/blob/main/.github/workflows/e2e.yml#L88-L90

Consider changing from 
 grep \"Agent 'bar' connection attempt 0: success\"" -> grep \"Connected to manager as  'bar'\""